README.md

```diff
@@ -37,7 +37,8 @@ please conduct a [compatability test](https://webauthn.firstyear.id.au/compat_te
 Known BROKEN Keys/Harwdare
 --------------------------
 
-* Pixel 3a / Pixel 4 + Chrome - Does not send correct attestation certificates, and ignores requested algorithms
+* Pixel 3a / Pixel 4 + Chrome - Does not send correct attestation certificates, and ignores requested algorithms.
+* Windows Hello + Windows 11 (22000.556) + Firefox running on AMD Ryzen 9 + MSI X570 Chipset (BIOS version A.F3, released 2021-09-27) - Truncates 16-byte authenticator aaguid to 1 byte.
```

Comment:
Could be worth saying "when aaguid is meant to be 16 bytes of 0, it emits a single 0 byte".